modules/charts/tsconfig.app.json

```diff
@@ -0,0 +1,30 @@
+{
+	"compilerOptions": {
+		"tsBuildInfoFile": "./node_modules/.tmp/tsconfig.app.tsbuildinfo",
```

Comment:
never send this, but reaching into node_modules seems like a smell. elaborate?

Reply:
temp folder for incremental build, likely not needed on a project this size, comes out of the box with vite and in this case node_modules works as it's meant to be transient